### ISSUE TITLE
Paint / Powder coat GAPS components

### ISSUE THREAD

**Issue by rocco8773** [GAPS-Collab/Tasks#192]

**Comment by Eclipsedclaw** [comment]
Is this the one that you mentioned in yesterday's ME call about powder coating task issue?

**Comment by rocco8773** [comment]
> Is this the one that you mentioned in yesterday's ME call about powder coating task issue?

Yes, this is the issue where I'm gathering all the things that need to be painted or powder coated.  For you, just keep to the powder coating items.  The painting items are happening here at UCLA.

**Comment by Eclipsedclaw** [comment]
I went through those gondola frame members in the CAD model. Are those already fabricated and come together with payload at Nevis, or they are waiting to be fabricated in CA or somewhere?

I'm asking this because I saw your email about the vendor we contacted for the past powder coating which is in CA. I'm trying to think whether it's better to find another vendor near NY

**Comment by rocco8773** [comment]
@Eclipsedclaw all parts are fabricated except for #196, and all the fabricated components should be at Nevis.  The boom side panel (#185) still needs to be drilled for the last RAT.  I also need to get you masking instructions for all the parts.  I'm actually a little naive about how to generate masking instructions, so could you ask you powder coating vendor what their preference is?

**Comment by Eclipsedclaw** [comment]
@rocco8773 okay I can ask them about the masking. I'll reach out to the original vendor first to see what they could offer.

**Comment by Eclipsedclaw** [comment]
I got reply from the vendor, painting of these 5 pieces would be roughly $180 - $210. And the approximately lead time would be 3 to 5 work days. This is not a official quote yet.

**Comment by rocco8773** [comment]
@Eclipsedclaw That doesn't seem all that bad.  Did they give any feedback on masking.  Also, I just discovered I still have #184 here at UCLA.

**Comment by Eclipsedclaw** [comment]
@rocco8773 They didn't give me masking info yet. I'll ask them tomorrow. I want to clarify that we want to mask every through holes, is that correct? 

Oh ok, once we decided to proceed with powder coating, I'll probably find people at Nevis and also pin you to ship to vendor? Maybe now it could stay at UCLA

**Comment by rocco8773** [comment]
> Oh ok, once we decided to proceed with powder coating, I'll probably find people at Nevis and also pin you to ship to vendor? Maybe now it could stay at UCLA

Sounds good to me.

> I want to clarify that we want to mask every through holes, is that correct?

At minimum we'll want to that.  We'll want to mask of the screw mating surfaces, which I'm assuming would be easier than just masking the holes.  We'll also want to mask of the RAT mating surfaces for #184 and #185.  My assumption, I'll make a parts drawing indicating the masked surfaces, then you can just mark it out with blue tape or a Sharpe.  In the end it won't need to be supper accurate.

**Comment by Eclipsedclaw** [comment]
The vendor replied with masking suggestion and it is basically special kapton tape masking. They accept drop off of the pieces. Now I'm asking them if they accept shipping options.

> For the holes, we have some plugs we can use just please mention it to the gentleman who receives them and show him what area. If you need to mask with tape, it would have to be done with high-heat tape. I'll link some below.
[https://www.grainger.com/product/15C510?gucid=N:N:PS:Paid:GGL:CSM-2295:4P7A1P:20501231&gclid=Cj0KCQjwuNemBhCBARIsADp74QTTI10AOcFRo9yf3cy1gMyCAGce01gsU_4Sk09rDFEVDtkpLHhdCzgaAjBCEALw_wcB&gclsrc=aw.ds](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.grainger.com%2Fproduct%2F15C510%3Fgucid%3DN%3AN%3APS%3APaid%3AGGL%3ACSM-2295%3A4P7A1P%3A20501231%26gclid%3DCj0KCQjwuNemBhCBARIsADp74QTTI10AOcFRo9yf3cy1gMyCAGce01gsU_4Sk09rDFEVDtkpLHhdCzgaAjBCEALw_wcB%26gclsrc%3Daw.ds&data=05%7C01%7Czeng.jia%40northeastern.edu%7C25d189f64ce1400cb03008db9a7cbdee%7Ca8eec281aaa34daeac9b9a398b9215e7%7C0%7C0%7C638273632784672639%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=LPcoXqG1G6NaqrjVETTwvRZTsKvzVhuYJhJHWSeOucU%3D&reserved=0)
You can find them at many stores and websites; this is just one example.

**Comment by Eclipsedclaw** [comment]
New updates:

- 2nd set of diagonals found, too long to bring back to NEU. Currently lay against thermal shelf at Nevis(see picture attached)
- GAPS-EBY-MEC-501 at UCLA
- GAPS-EBY-MEC-201 can't find
- GAPS-FRM-MEC-020 found, too long to bring back to NEU. Currently lay against thermal shelf at Nevis(see picture attached)
- GAPS-FRM-MEC-061 didn't find

Might have to mask them at Nevis
![WechatIMG1130](https://github.com/GAPS-Collab/Tasks/assets/37788723/0b76c066-fc2b-4df8-9378-2036cecb1bc4)
 and schedule a pickup delivery

**Comment by rocco8773** [comment]
20231018: All parts located at Nevis and UCLA were shipped to Melrose Metal Finishing for powder coating.
